utils/TimeMachineUtils.java

```diff
@@ -36,7 +36,12 @@
 
     public Vocab(String[][] tokens, int minFreq, String[] reservedTokens) {
         // Sort according to frequencies
-        LinkedHashMap<String, Integer> counter = countCorpus2D(tokens);
+        LinkedHashMap<Object, Integer> counterObject = countCorpus2D(tokens);
```

Comment:
use wildcard here
```suggestion
        LinkedHashMap<?, Integer> counterObject = countCorpus2D(tokens);
```

Reply:
This is fixed with 14.3 changes so once that is merged I can update changes on this branch